### PR TITLE
chore: cleanup of redis implementation of sets/hashmaps

### PR DESCRIPTION
**Pull request by romange**

### CODE REVIEW

File: src/server/set_family.cc [hunk 1]
```diff
@@ -1627,7 +1493,7 @@ bool SetFamily::ConvertToStrSet(const intset* is, size_t expected_len, robj* des
   char buf[32];
   int ii = 0;
 
-  if (GetFlag(FLAGS_use_set2)) {
+  if (true) {
```
Comment (kostasrim) [inline src/server/set_family.cc:1496]: ??

File: src/server/set_family.cc [hunk 2]
```diff
@@ -72,33 +68,12 @@ intset* IntsetAddSafe(string_view val, intset* is, bool* success, bool* added) {
...
-}
-
 pair<unsigned, bool> RemoveStrSet(uint32_t now_sec, ArgSlice vals, CompactObj* set) {
   unsigned removed = 0;
   bool isempty = false;
-  auto* shard = EngineShard::tlocal();
+  DCHECK(IsDenseEncoding(*set));
 
-  if (IsDenseEncoding(*set)) {
+  if (true) {
```
Comment (kostasrim) [inline src/server/set_family.cc:76]: same here?

File: src/server/rdb_load.cc [hunk 3]
```diff
@@ -502,36 +499,24 @@ void RdbLoaderBase::OpaqueObjLoader::CreateSet(const LoadTrace* ltrace) {
...
      * to avoid rehashing */
-    if (len > DICT_HT_INITIAL_SIZE && !resizeStringSet(res, len, use_set2)) {
+    if (len > 2 && !resizeStringSet(res, len)) {
       LOG(ERROR) << "OOM in dictTryExpand " << len;
       ec_ = RdbError(errc::out_of_memory);
       return;
     }
 
-    if (use_set2) {
+    if (true) {
```
Comment (kostasrim) [inline src/server/rdb_load.cc:519]: and in other places as well :)

**Review by kostasrim — APPROVED** [review]

File: src/server/set_family.cc (hunk 1 above)
Comment (romange) [reply src/server/set_family.cc:1496]: Otherwise diffs become annoying due to indentation shifts.

**Review by kostasrim — APPROVED** [review]

**Pull request merged**